experimental/sdk/webpubsub-socketio-extension/src/SIO/components/negotiate.ts

```diff
@@ -0,0 +1,168 @@
+import {
+  debugModule,
+  AzureSocketIOOptions,
+  AzureSocketIOCredentialOptions,
+  NegotiateOptions,
+  getWebPubSubServiceClient,
+} from "../../common/utils";
+import * as SIO from "socket.io";
+import { IncomingMessage, ServerResponse } from "http";
+
+import {
+  EXPRESS_SESSION_COOKIE_NAME,
+  NEGOTIATE_HANDLER_PROPERTY_NAME,
+  WEB_PUBSUB_OPTIONS_PROPERY_NAME,
+} from "./constants";
+import { Store } from "express-session";
+import * as cookieParser from "cookie";
+import { Crypto } from "../../common/encrypt";
+import { randomBytes } from "crypto";
+import { parse } from "url";
+
+const debug = debugModule("wps-sio-ext:SIO:negotiate");
+
+// Reference: https://github.com/socketio/socket.io/blob/4.6.2/lib/namespace.ts#L22
+interface ExtendedError extends Error {
```

Comment:
Can it be directly import as `import { ExtendedError } from "socket.io/dist/namespace";`

Reply:
It works. updated.